docs/installation.md

```diff
@@ -11,36 +11,67 @@ pipx works on macOS, linux, and Windows.
 
 ## Installing pipx
 
-On macOS:
+#### On macOS:
```

Comment:
Why h4 instead of h3? That level seems available.

Reply:
Fixed